src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: no-repeat center center;
+  background-image: url(./images/star.svg);
```

Comment:
You can add it to `background` too, but it's not necessary 
https://developer.mozilla.org/en-US/docs/Web/CSS/background#syntax